x-pack/plugins/apm/server/lib/backends/get_throughput_charts_for_backend.ts

```diff
@@ -41,6 +39,12 @@ export async function getThroughputChartsForBackend({
     offset,
   });
 
+  const { intervalString, bucketSize } = getBucketSize({
+    start: startWithOffset,
+    end: endWithOffset,
+    minBucketSize: 60,
+  });
```

Comment:
Slightly off-topic: I think `bucketSize` and `intervalString` both stores the same interval in seconds, only difference being that `bucketSize` is a number (`5`) and `intervalString` is a string (`5s`).

Would be great if the names reflected this, a couple of suggestions:
 - `bucketSize` / `bucketSizeString`
 - `interval` / `intervalString`
 - `intervalNumber`/ `intervalString`

Another approach would be an object like: `bucketSize.asNumber()` and `bucketSize.asString()`

(I kinda like the last one because then both representations are co-located)

Reply:
Yeah, that's a good idea. Although I'd rather changing it on another PR, if that's okay for you.

Reply:
Sure 👍